site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -312,6 +313,33 @@ If managed FusionAuth hosting does not meet your needs, open a link:/contact/[co
 
 For instance, if you need longer retention of database backups, we're happy to discuss that with you.
 
+== SLAs and Availability
+
+We know that availability of your auth system is critical. Below are some of the steps FusionAuth takes to ensure that FusionAuth cloud systems are available. You can find information about service level agreements (SLAs) in the link:/license/[FusionAuth license] and link:/license-faq/[license FAQ].
+
+For all deployments:
+
+* All components are monitored via external software systems as well as internal metrics gathering systems.
+* The application architecture is not exotic, reducing risk. This is a three tier web application, which is a well understood architecture.
+* External security researchers are encouraged to submit security issues via a bug bounty program.
+* The FusionAuth software and infrastructure is pentested on a regular basis.
+* Major changes, such as FusionAuth upgrades or operating system upgrades, are scripted and the timing of changes is controlled by the customer.
+* Modifications to the underlying product are tested extensively, including load testing where appropriate.
+* We have automated rulesets to protect against DDOS.
+* We use a world class cloud provider's managed services for system components where appropriate.
+* Network firewalls are automatically configured on deployment to set up "least privileged" access to each system component.
+
+For high availability deployments:
+
+* Each deployment has redundant components. This includes the DNS, load balancer, compute nodes running FusionAuth, and the relational database.
```

Comment:
And Elasticsearch. Not sure you have to mention it by name, or at all.

Reply:
I modified to implicitly include elastic search.